edn/src/types.rs

```diff
@@ -214,12 +264,6 @@ impl Value {
     }
 }
 
-impl From<f64> for Value {
```

Comment:
One type can have many `From` implementations.  If anything, I want `edn::Value` to have `From` for _more_ types (when there's no ambiguity).  Can you explain why you removed this?